themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Introducing the Puluminaries"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-05-02T08:04:47-05:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: We are very excited to announce and showcase our new program of community champions, the Puluminaries! 
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - matt-stratton
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - community
+aliases:
+    - /puluminaries/
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Pulumi is more than a way to build, deploy, and manage your infrastructure and cloud applications. Pulumi is also a strong and vibrant community. We are very excited to announce and showcase our new program of community champions, the Puluminaries!
+
+<!--more-->
+<img src="puluminaries.png" width="500" height="500" alt="Engin Diri"/>
+
+The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure. The program is open to all members of the community, and we welcome anyone to join the program.
+
+While the Puluminaries have been active in the community for quite some time, we have officially launched the program at [PulumiUP](https://pulumi.com/pulumi-up) in 2022, and it is our pleasure to introduce the first cohort of members to the program!
```

Comment:
While the Puluminaries themselves have been active... I think sounds better and refers to them not as a corporate entity but rather individuals. 

i don't think you need the "have" in "we have officially launched".